apertium-kaa.kaa.lexc

```diff
@@ -31158,57 +31158,40 @@ Abdulǵaniyev:Abdulǵaniyev  NP-COG-MF ; !
 Abdulganiyeva:Abdulganiyeva  NP-COG-MF ; !  
 Abdulǵaniyeva:Abdulǵaniyeva  NP-COG-MF ; !  
 Abdurahmanov:Abdurahmanov  NP-COG-OV ; !  
-Abdurahmanova:Abdurahmanova  NP-COG-OV ; !  
 Abduraimov:Abduraimov  NP-COG-OV ; !  
-Abduraimova:Abduraimova  NP-COG-OV ; !  
 Abe:Abe  NP-COG-MF ; !  
 Abibullayev:Abibullayev  NP-COG-MF ; !  
 Abibullayeva:Abibullayeva  NP-COG-MF ; !  
 Abilov:Abilov  NP-COG-OV ; !  
-Abilova:Abilova  NP-COG-OV ; !  
 Abisov:Abisov  NP-COG-OV ; !  
-Abisova:Abisova  NP-COG-OV ; !  
 Abisovich:Abisovich  NP-PAT-VICH ; !  
```

Comment:
I think the right side of these shouldn't have the "ich" because it needs to add the "na" also.